src/wallet/spend.cpp

```diff
@@ -105,6 +105,61 @@ void CoinsResult::clear()
     other.clear();
 }
 
+struct SelectedInputs
+{
+    std::set<COutput> coins;
+    // If subtract fee from outputs is active, the 'total_amount'
+    // will be the sum of each output effective value
+    // instead of the sum of the outputs amount
```

Comment:
In 4946384e2f56333274baf7fab1dfd41b046e12a1 "wallet: encapsulate pre-selected-inputs lookup into its own function"

This comment is backwards. The total amount uses the effective value normally, and real value when SFFO.

Reply:
ups, pushed.